### PR TITLE
Created Marshall_Repeating_Turn

### PR DESCRIPTION
**Pull request by youraveragedeveloper7**
Created marshall_repeating_turn function and tweaked a bit of the marshall_guess function

### CODE REVIEW

File: source/CardGames.py [hunk 1]
```diff
@@ -636,7 +645,37 @@ def fugitive_repeating_turn(self):
...
+      while True:
+        try:
+          deck = int(input("\nSelect which deck to draw two cards from ['1' (1-14), '2' (15-28), '3' (29-42)] and press enter... "))
+        except ValueError:
+          print("Invalid input, try again.")
+          continue
+        if deck in [1, 2, 3]:
+          break
+      
+      match deck:
```
Comment (SCustodero) [inline source/CardGames.py:660]: Woah this is fancy! Looks great, also I really like the changes to the print outs. Good stuff

**Review by SCustodero — APPROVED** [review]

File: source/CardGames.py [hunk 2]
```diff
@@ -636,7 +645,37 @@ def fugitive_repeating_turn(self):
...
+          deck = self.MidRangeDeck
+        case 3: 
+          deck = self.HighRangeDeck
+      
+      #Draw a card from selected deck
+      self.marshall.addCard(deck.pop(), isKnown=True)
+
+      #Show marshall's hand, including drawn card
+      print("\nThis is your hand:")
+      self.marshall.showHand()
```
Comment (georgiok741) [inline source/CardGames.py:673]: I am glad you are showing the hand and board after each turn! Good touch brother!

File: source/CardGames.py (hunk 1 above)
Comment (youraveragedeveloper7) [reply source/CardGames.py:660]: Thank you brother! I appreciate it.

File: source/CardGames.py (hunk 2 above)
Comment (youraveragedeveloper7) [reply source/CardGames.py:673]: Thank you brotha! Glad you liked the polishing.

**Review by georgiok741 — APPROVED** [review]

**Pull request merged**